1.0/docs/devguide/utility-functions.md

```diff
@@ -54,13 +54,13 @@ provides a set of useful convenience functions for instances to use.
 
 *   `isDebouncerActive(jobName)`. Returns true if the named debounce task is waiting to run.
 
-*   `transform(transform, node)`. Applies a CSS transform to the specified node,
-    or this element if no node is specified.
+*   `transform(transform, [node])`. Applies a CSS transform to the specified node,
```

Comment:
It's Google's JS doc style to use `opt_Node`. `[node]` can be confused for an array argument.

@arthurevans we should probably rely on the API docs for these methods: http://polymer.github.io/polymer/